js/src/forum/components/SignUpModal.js

```diff
@@ -161,7 +161,7 @@ export default class SignUpModal extends Modal {
 
     const body = this.submitData();
 
-    app
+    return app
```

Comment:
Wouldn’t this be pointless/need more changes since the window is reloaded? Any then statement after wouldn’t be able to run.